.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
+profile=black
```

Comment:
This is so that `black` doesn't have to cleanup after `isort`. I'm not sure whether you've noticed this happening.